pyproject.toml

```diff
@@ -0,0 +1,73 @@
+[tool.poetry]
+name = "pyndl"
+version = "0.8.3"
+description = "Naive discriminative learning implements learning and classification models based on the Rescorla-Wagner equations."
+
+license = "MIT"
+
+authors = ["Konstantin Sering <konstantin.sering@uni-tuebingen.de>",
+           "Marc Weitz",
+           "David-Elias Künstle",
+           "Lennard Schneider",
+           "Elnaz Shafaei-Bajestan"
+           ]
+
+readme = "README.rst"  # Markdown files are supported
+
+repository = "https://github.com/quantling/pyndl"
+homepage = "https://pyndl.readthedocs.io/en/latest/"
+
+classifiers = ['Development Status :: 4 - Beta',
+               'Environment :: Console',
+               'Intended Audience :: Science/Research',
+               'Operating System :: POSIX :: Linux',
+               'Operating System :: MacOS',
+               'Topic :: Scientific/Engineering',
+               'Topic :: Scientific/Engineering :: Artificial Intelligence',
+               'Topic :: Scientific/Engineering :: Information Analysis',]
+
+build = "build.py"
+
+[tool.poetry.dependencies]
+python = ">=3.8,<3.10"  # Compatible python versions must be declared here
```

Comment:
Does anything breaks for Python 3.10? Or why are we restricting python to below 3.10?